finagle/h2/src/main/scala/com/twitter/finagle/buoyant/h2/netty4/UnpoolHandler.scala

```diff
@@ -0,0 +1,56 @@
+package com.twitter.finagle.buoyant.h2
+package netty4
+
+import io.netty.buffer.{ByteBuf, ByteBufHolder, EmptyByteBuf, Unpooled}
+import io.netty.channel.ChannelHandler.Sharable
+import io.netty.channel.{ChannelHandlerContext, ChannelInboundHandlerAdapter}
+
+/**
+ * This was originally copied from finagle's implementation of AnyToHeapInboundHandler:
```

Comment:
This seems like the right thing to do for now.  Is there a ticket for figuring out buffer management in the long term?